src/VisualStudio/LiveShare/Impl/AbstractGoToDefinitionWithFindUsagesServiceHandler.cs

```diff
@@ -39,7 +39,7 @@ public AbstractGoToDefinitionWithFindUsagesServiceHandler(IMetadataAsSourceFileS
 
         public async Task<object> HandleAsync(LSP.TextDocumentPositionParams request, RequestContext<Solution> requestContext, CancellationToken cancellationToken)
         {
-            var document = _solutionProvider.GetDocument(request.TextDocument);
+            var document = _solutionProvider.GetDocument(request.TextDocument, null);
```

Comment:
why can this one not use the request context document?

Reply:
This is a LiveShare handler, the RequestContext is not ours, and this doesn't look like it goes through our LSP server.